packages/ruby/lib/readme/har/response_serializer.rb

```diff
@@ -30,14 +26,45 @@ def as_json
 
       private
 
+      def content
+        if response_body.nil?
+          empty_content
+        elsif @response.content_type == "application/json"
```

Comment:
You might want to add support for JSON variants like `application/x-json`, `text/json`, or vendored JSON content types like `application/vnd.github.v3.star+json`.

I added something similar last week in one of our core OpenAPI tooling methods last week here: https://github.com/readmeio/oas/blob/master/packages/tooling/src/operation.js#L62

Reply:
Good call. Since this PR is focused on empty response bodies and only moved the JSON code, I'm going to implement your suggestion in a separate feature. I've created a [Trello card](https://trello.com/c/wFCZ1BKN/27-support-multiple-json-mime-types) to keep track of it.